<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>570</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>40</y>
      <width>20</width>
      <height>400</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>840</width>
      <height>10</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>611</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>399</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>90</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QPushButton" name="pb_updateFile">
       <property name="text">
        <string>上传文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_downloadFile">
       <property name="text">
        <string>下载文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_clearChatInfo">
       <property name="text">
        <string>清空聊天</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pb_send">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>440</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QListView" name="lv_chatRoom">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="lv_user">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="lv_file">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>121</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lb_userList">
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="text">
        <string>用户列表  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_preFixCurTime">
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>当前时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_curTime">
       <property name="text">
        <string>NULL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_preFixOnlineNums">
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>在线人数：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_onlineNums">
       <property name="text">
        <string>NULL</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>textEdit</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>lv_chatRoom</zorder>
   <zorder>lv_user</zorder>
   <zorder>pb_send</zorder>
   <zorder>lv_file</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
